src/domain/events/__tests__/EventsPage.test.tsx

```diff
@@ -323,14 +319,12 @@ test('renders events list and load more events button works', async () => {
     ]
   `);
 
-  expect(consoleErrorSpy.mock.calls).toMatchInlineSnapshot(`
+  expect(consoleErrorSpy.mock.calls.at(-1)).toMatchInlineSnapshot(`
```

Comment:
There were now two errors, first one was actually a deprecation warning, so just checking the last one here for the expected "Failed to fetch more events" errors.